index.html

```diff
@@ -0,0 +1,9 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Seyda's Bakes</title>
```

Comment:
Great title bro

Reply:
why is the body so far down ???

Reply:
Dunno about you, but my body is below my head....